e2e/playwright/actions/src/tests/create-folder.spec.ts

```diff
@@ -23,65 +23,73 @@
  */
 
 import { expect } from '@playwright/test';
-import { ApiClientFactory, Utils, folderErrors, getUserState, test } from '@alfresco/playwright-shared';
+import {
+  AdfFolderDialogComponent,
+  ApiClientFactory,
+  DataTableComponent,
+  LoginPage,
+  NodesApi,
+  Utils,
+  errorStrings,
+  test
+} from '@alfresco/playwright-shared';
+import { logger } from '@alfresco/adf-cli/scripts/logger';
 
-test.use({ storageState: getUserState('hruser') });
 test.describe('Create folders', () => {
   const apiClientFactory = new ApiClientFactory();
+  let nodesApi: NodesApi;
+  let folderDialog: AdfFolderDialogComponent;
+  let folderTable: DataTableComponent;
   let randomFolderName: string;
   let randomFolderTitle: string;
   let randomFolderDescription: string;
+  const dialogString = 'dialog';
+  const createNewFolderString = 'Create new folder';
+  const spacesString = '   ';
   const commonFolderName = `playwright-folder-${Utils.random()}`;
-  let folderId: string;
+  const username = `user-${Utils.random()}`;
 
   test.beforeAll(async () => {
-    await apiClientFactory.setUpAcaBackend('hruser');
-    const node = await apiClientFactory.nodes.createNode('-my-', { name: commonFolderName, nodeType: 'cm:folder' });
-    folderId = node.entry.id;
+    try {
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username });
+      nodesApi = await NodesApi.initialize(username, username);
+      await nodesApi.createFolder(commonFolderName);
+    } catch (error) {
+      logger.error(`beforeAll failed : ${error}`);
+    }
   });
 
-  test.beforeEach(async ({ personalFiles }) => {
+  test.beforeEach(async ({ personalFiles, page }) => {
     randomFolderName = `playwright-folder-${Utils.random()}`;
     randomFolderTitle = `folder-title-${Utils.random()}`;
     randomFolderDescription = `folder-description-${Utils.random()}`;
-    await personalFiles.navigate();
+    folderDialog = personalFiles.folderDialog;
+    const loginPage = new LoginPage(page);
+    try {
+      await loginPage.loginUser(
+        { username: username, password: username },
```

Comment:
can be just 
{ username, password: username }

no need to write the same at left and right side if both are the same

Reply:
updated, thanks